Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
+  const durationInHours = tasks.map(x => x.duration / 60);
+  const filterHours = durationInHours.filter(x => x >= 2);
+  for (let i = 0; i < filterHours.length; i++) {
```

Comment:
Can you think of an array method (`forEach`, `map`, `filter` or `reduce`) that can replace this `for` loop?